FunctionalPreprocessing/FunctionalPreprocessing.XNAT_MARK_RUNNING_STATUS

```diff
@@ -50,11 +50,15 @@ get_options()
 	local arguments=($@)
 
 	# initialize global output variables
+    unset g_user
+    unset g_password	
+    unset g_server
 	unset g_project
```

Comment:
We should settle between the two of us on an indentation mechanism. 

Should we use tabs or spaces? I prefer tabs. But I don't feel overly strong about this. You may find some files in which I've used tabs and others in which I've used spaces. I looks like this file was previously all indented using tabs, but at least in some cases in this file, you've used spaces. For example, this `unset g_user` line is indented with spaces, while a few lines down, the `unset g_project` is indented with a tab. 

As I said, I don't feel strongly enough about this to insist we do it one way or the other, but I think we should pick a style and then start using it consistently. If you're OK with using tabs, then please convert this file to using all tabs for indentation. If you prefer to use spaces, then convert this file to using all spaces for indentation. Being consistent will make file comparisons much easier and make sure things line up nicely no matter what editor is being used.

Reply:
tab 👍